app/assets/stylesheets/application.scss

```diff
@@ -102,6 +101,19 @@ tr input[type=text] {
   margin-bottom: 3px;
 }
 
+.btn-margin {
+  margin-right: 50px;
+}
+
+.dash-btn {
+  width: 50%;
+  margin: 15px 0 ;
+}
+
+.bordered-div {
+  border: 1px solid #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.